packages/ui/src/components/Forms/Checkbox/Checkbox.tsx

```diff
@@ -56,7 +58,7 @@ export const Checkbox = memo(forwardRef<HTMLInputElement, CheckboxProps>(({
 		value,
 	}, forwardedRef)
 
-	const { className, indeterminate, ...nativeInputProps } = props
+	const { className, ...nativeInputProps } = omit(props, ['indeterminate'])
```

Comment:
BETTER:

- cleaner — does not waste screen estate as aliasing to e.g. `INTENTIONALLY_UNUSED_INDETERMINATE`, array of omitted can be larger
- less hesitation to keep it unused and somebody might accidentally remove unused and cause error
- lint ready: We can set up some listing rules to catch up unused vars/args/imports and this will not clog up the log.

SAME:
- It still means "I intend to omit this", is type safe

Reply:
few lines bellow, you use `props.indeterminate` so you can replace it with `indeterminate`, so it won't be unused.

Reply:
Removed.